Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md

```diff
@@ -1,74 +1,245 @@
-Use the Microsoft Teams integration to send messages and notifications to your team members and create meetings.
-This integration was integrated and tested with version 1.0 of Microsoft Teams.
+Use the Microsoft Teams integration to send messages, notifications, and create meetings.
+This integration was tested with version 1.0 of Microsoft Teams.
 
-**Note:**
+This document includes the following sections to help you understand, set up, and use the integration effectively:
 
-- This integration is supported in Cortex XSOAR 8 and up and Cortex XSIAM without using an engine.
-- The integration has the ability to run built-in Cortex XSOAR commands, through a mirrored channel. Make sure to pass the command in the chat exactly as typed in the CORTEX XSOAR CLI. For example: `!DeleteContext all=yes`. Use the command `mirror-investigation` to mirror/create a mirrored channel.
-- For use cases where it is only needed to send messages to a specific channel, we recommend checking the [Microsoft Teams via Webhook Integration](https://xsoar.pan.dev/docs/reference/integrations/microsoft-teams-via-webhook), which has a simpler setup.
+1. [Integration Architecture](#integration-architecture)
+2. [Setup and Configuration](#setup-and-configuration)
+3. [Setup Examples](#setup-examples)
+3. [Known Limitations](#known-limitations)
+4. [Important Information](#important-information)
+5. [Troubleshooting](#troubleshooting)
+6. [Download Demisto Bot](#download-demisto-bot)
+7. [Commands](#commands)
 
-## Integration Architecture
+# Integration Architecture
 
-Data is passed between Microsoft Teams and Cortex XSOAR through the bot that you will configure in Microsoft Teams. A webhook (that you will configure) receives the data from Teams and passes it to the messaging endpoint. The web server on which the integration runs in Cortex XSOAR listens to the messaging endpoint and processes the data from Teams. You can use an engine for communication between Teams and the Cortex XSOAR server. In order to mirror messages from Teams to Cortex XSOAR, the bot must be mentioned, using the @ symbol, in the message.
```

Comment:
Isn't this also for XSIAM? If so, lets change it to Cortex XSOAR/XSIAM. Also lets call it the Cortex XSOAR/XSIAM tenant and not server.